gaas-ui/package.json

```diff
@@ -14,81 +14,88 @@
     "jsonwebtoken": "^8.5.1",
     "material-ui-dropzone": "^3.5.0",
-    "material-ui-image": "^3.3.0",
+    "material-ui-image": "^3.3.2",
     "npm-check-updates": "^9.1.0",
     "react": "^16.14.0",
     "react-dom": "^16.14.0",
     "react-json-view": "^1.19.1",
-    "react-router": "^5.2.0",
-    "react-router-dom": "^5.2.0",
+    "react-router": "^5.2.1",
+    "react-router-dom": "^5.3.0",
     "react-scripts": "^4.0.0",
     "react-testing-library": "latest",
     "react-toastify": "^6.1.0",
-    "reaviz": "^10.1.0",
-    "sanitize-html": "^2.4.0",
+    "reaviz": "^10.4.4",
+    "regenerator-runtime": "^0.13.9",
+    "sanitize-html": "^2.5.2",
     "statuses": "^2.0.1",
     "validator": "^13.6.0"
   },
   "devDependencies": {
-    "@babel/preset-env": "^7.12.7",
-    "@babel/preset-react": "^7.12.7",
-    "@testing-library/react": "^11.2.2",
-    "@types/body-parser": "^1.19.0",
-    "@types/enzyme": "^3.10.8",
+    "@babel/preset-env": "^7.15.8",
+    "@babel/preset-react": "^7.14.5",
+    "@testing-library/react": "^11.2.7",
+    "@types/body-parser": "^1.19.1",
+    "@types/core-js": "^2.5.5",
+    "@types/enzyme": "^3.10.10",
     "@types/enzyme-adapter-react-16": "^1.0.6",
-    "@types/express": "^4.17.9",
-    "@types/jest": "26.0.15",
-    "@types/morgan": "^1.9.2",
-    "@types/react-router-dom": "^5.1.6",
-    "@types/sanitize-html": "^2.3.2",
+    "@types/express": "^4.17.13",
+    "@types/jest": "^27.0.2",
+    "@types/morgan": "^1.9.3",
+    "@types/react-router-dom": "^5.3.2",
+    "@types/regenerator-runtime": "^0.13.1",
+    "@types/sanitize-html": "^2.5.0",
     "@types/statuses": "^2.0.0",
-    "@types/supertest": "^2.0.10",
-    "@types/webpack": "^4.41.25",
-    "@typescript-eslint/eslint-plugin": "^4.9.1",
-    "@typescript-eslint/parser": "^4.9.1",
-    "accessibility-checker": "^3.1.6",
-    "axios-mock-adapter": "^1.19.0",
+    "@types/supertest": "^2.0.11",
+    "@types/webpack": "^4.41.31",
+    "@typescript-eslint/eslint-plugin": "^4.33.0",
+    "@typescript-eslint/parser": "^4.33.0",
+    "accessibility-checker": "^3.1.11",
+    "axios-mock-adapter": "^1.20.0",
+    "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.1.0",
+    "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-plugin-transform-es2015-modules-commonjs": "^6.26.2",
     "concurrently": "^5.3.0",
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.5",
-    "enzyme-to-json": "^3.6.1",
+    "enzyme-to-json": "^3.6.2",
     "eslint-config-react-app": "^5.2.1",
-    "eslint-plugin-flowtype": "^5.2.0",
-    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-flowtype": "^5.10.0",
+    "eslint-plugin-import": "^2.25.2",
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-notice": "^0.9.10",
-    "eslint-plugin-react": "^7.21.5",
+    "eslint-plugin-react": "^7.26.1",
     "eslint-plugin-react-hooks": "^4.1.2",
     "fetch-mock": "^9.11.0",
     "morgan": "^1.10.0",
     "node-sass": "^4.14.1",
```

Comment:
I am getting weird issues unless this is set as `^6.0.1` on Ubuntu

Reply:
@rocky341 is this repeatable or just my environment being weird?

Reply:
When I say "this" I mean `node-sass`

Reply:
@t92549 what kind of issues? Seems to be running fine on my machine. I updated node-sass locally to 6.0.1 and that also seems to run fine for me. I'm happy to change node-sass to 6.0.1 then if you would prefer?

Reply:
It's okay, it works fine for node 14 just had issues when using node 16. Also lots of warnings for both but these can be cleaned up later